setup.py

```diff
@@ -25,6 +27,8 @@
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
```

Comment:
I always add `python_requires` and `project_urls` as a matter of habit if it's missing.

Reply:
Thank you for adding them. I've not yet developed that habit and always forget to add them.